lib/features/sks-menu/presentation/widgets/sks_menu_loading.dart

```diff
@@ -0,0 +1,70 @@
+import "package:flutter/material.dart";
+import "package:lottie/lottie.dart";
+
+import "../../../../../theme/app_theme.dart";
+import "../../../../../utils/context_extensions.dart";
+import "../../../../gen/assets.gen.dart";
+
+class LottieLoading extends StatefulWidget {
+  const LottieLoading({
+    super.key,
+  });
+
+  @override
+  _LottieLoadingState createState() => _LottieLoadingState();
+}
+
+class _LottieLoadingState extends State<LottieLoading> {
+  final bool _showOldMenu = false;
+
+  void _gotoOldMenu() {
+    //Navigator.push(
+    //context,
+    //MaterialPageRoute(builder: (context) => const _SksMenuView()),
+    // );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Column(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: [
+          const Spacer(),
+          Center(
+            child: SizedBox(
+              width: 200,
+              height: 200,
+              child: Lottie.asset(
```

Comment:
Lottie here could be a little more customized. Ie. cache, frame rate. Please see reference in file used somewhere else: 

![image](https://github.com/user-attachments/assets/8d8a70fe-11d8-4675-bfa4-229dce57a10d)